### DISCUSSION TITLE
Need help making RPM

### DISCUSSION THREAD

**Discussion by GitMeTheGoods** [community/community#21946] [New to GitHub]
<p>Hi all, I am new to github and linux as well. I am trying to set up a leak proof VPN on qubes os  3.2. I am using this repository:</p>
<p><a href="https://github.com/Rudd-O/qubes-vpn" rel="nofollow noopener">https://github.com/Rudd-O/qubes-vpn</a></p>
<p>I have cloned the repository into a vm under my Fedora-25 template. Since I am new to qubes and github I am having a hard time with this step:</p>
<p>Make the RPM on the folder of your clone make rpm</p>
<p>It says make an RPM on the folder of my clone. Does this mean I am supposed to turn the folder itself into a RPM? Or am I supposed to make the spec file an RPM?</p>
<p>Also, how would I do this? Just entering the command make rpm gives me an error:</p>
<p>make: *** No rule to make target ‘rpm’.  Stop.</p>
<p>I am lost so I tried:</p>
<p>[user@personal-vm-use Downloads]$ sudo make rpm -qubes-vpn</p>
<p>and I see:</p>
<p>make: invalid option – ‘u’<br>
make: invalid option – ‘-’</p>
<h1>GNU Make 4.1</h1>
<p>and I see a bunch going on in the command prompt and ultimately see:</p>
<h1>Not a target:</h1>
<p>rpm:</p>
<h1>Command line target.</h1>
<h1>Implicit rule search has not been done.</h1>
<h1>Modification time never checked.</h1>
<h1>File has not been updated.</h1>
<p>I’m not exactly sure what commands I should be entering to make the cloned repository a rpm. And I am not sure if I am supposed to make the actual folder a rpm like the way I am interpreting the guide, or am I supposed to make the .spec file the rpm?</p>
<p>Any help would be appreciated. Thanks.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you’re having a problem with the documentation of a specific project being unclear, the best way to address it is to first check for the presence of a CONTRIBUTING guide and follow any instructions there. Since this project doesn’t have a CONTRIBUTING guide, I checked the README for suggestions and found this at the bottom:</p>
<blockquote>
<p>File issues on this project if you could not get it to work, or there are errors in the software or the documentation.</p>
</blockquote>
<p>So I would recommend <a href="https://github.com/Rudd-O/qubes-vpn/issues" rel="nofollow noopener">filing an issue</a> on the project asking for assistance.</p>
<p>I hope that helps 😀</p>

**Comment by GitMeTheGoods** [comment]
<p>Good idea, thank you.</p>
